docs/api/en/extras/PMREMGenerator.html

```diff
@@ -15,7 +15,36 @@ <h1>[name]</h1>
 			CubeUV format that allows us to perform custom interpolation so that we can support nonlinear formats such as RGBE.
 			Unlike a traditional mipmap chain, it only goes down to the LOD_MIN level (above), and then creates extra even more
 			filtered 'mips' at the same LOD_MIN resolution, associated with higher roughness levels. In this way we maintain
-			resolution to smoothly interpolate diffuse lighting while limiting sampling computation.
+			resolution to smoothly interpolate diffuse lighting while limiting sampling computation.<br/><br/>
+
+			Note: The minimum [page:MeshStandardMaterial]'s roughness depends on the size of the provided texture:
```

Comment:
You might also note this is a slightly conservative bound. If your render has small dimensions or the shiny parts have a lot of curvature, you may still be able to get away with a smaller texture size.

Reply:
Added that additional info verbatim.